biogeochem/EDPhysiologyMod.F90

```diff
@@ -686,7 +713,12 @@ subroutine phenology_leafonoff(currentSite)
                    ! add falling leaves to litter pools . convert to KgC/m2                    
                    currentCohort%leaf_litter = currentCohort%bl  
                    currentCohort%bl          = 0.0_r8                                        
```

Comment:
@xuchongang , see here, if dstatus is 1, then leaves are dropped, so bl is zero, and there wont be any drought amplified maintenance turnover...  Am I missing something?

Reply:
dry amplified maintenace turnover is for evergreen trees.